src/raftstore/store/store.rs

```diff
@@ -287,6 +293,18 @@ impl<T, C> Store<T, C> {
         Ok(())
     }
 
+    fn clean_up_meta(&mut self, wb: &mut WriteBatch, region: &metapb::Region) {
```

Comment:
`clean_up_meta` here confuses me, why not naming clear directly? and this function seems not only clearing the meta but also writing a tombstone key.